Symfony/CS/Fixer/PSR2/FunctionCallSpaceFixer.php

```diff
@@ -121,11 +121,13 @@ private function getFunctionyTokens()
     }
 
     /**
-     * Gets the name of tokens that are actually language construction.
+     * Gets the token kinds of actually language construction.
      *
+     * @var int[] $languageConstructionTokens Token kinds.
```

Comment:
I thought this was phpdoc for a property you see. You just need to move that annotation inside the method itself as seperate phpdoc.